React/my-app/src/redux/accounts/actions.js

```diff
@@ -16,11 +19,13 @@ export const verifyUser = body => {
           type: actions.CHECK_USER_SUCCESS,
           payload: { token: response.data.access_token }
         });
+      } else {
+        throw INVALID_USER;
```

Comment:
the standard way to throw and handle exceptions is using the Error object.
so
`throw new Error(message);` is that you have to do